gr-qtgui/include/gnuradio/qtgui/eyedisplayform.h

```diff
@@ -0,0 +1,141 @@
+/* -*- c++ -*- */
+/*
+ * Copyright 2011,2012,2020 Free Software Foundation, Inc.
```

Comment:
since this is a new file, it should only have a copyright of 2020, please check all other files for this on this branch